content/en/docs/instrumentation/net/exporters.md

```diff
@@ -105,66 +104,28 @@ AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport
 
 If you are using .NET 5 or higher, the previous code sample is not required.
 
-## Jaeger
+### Jaeger
 
 If you are using [Jaeger](https://www.jaegertracing.io/) to visualize trace
```

Comment:
`If you are using [Jaeger](https://www.jaegertracing.io/) to visualize trace data, this is how to run it in a docker container:`

This sentence is confusing (you use jaeger but we ask you to install jaeger?)

I suggest something like:

`To try out the OTLP exporter, you can run  [Jaeger](https://www.jaegertracing.io/) as an OTLP endpoint and for trace visualization in a docker container`